magento/pwa-studio
# Developer can style content types output differently per viewport

:heavy_exclamation_mark::heavy_exclamation_mark:DO NOT MERGE BEFORE https://github.com/magento/magento2-page-builder/issues/558 closed :heavy_exclamation_mark::heavy_exclamation_mark:
## Description
Convert style blocks to inline styles to prevent BIC changes

## Related Issue
https://jira.corp.magento.com/browse/PB-558
magento/magento2-page-builder#558

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to any page with Page Builder content.
2. Verify that content renders as before.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
